lib/config.py

```diff
@@ -192,6 +192,10 @@ def _setup_config_parser_options(self):
         self.parser.add_argument('--log-size',
                                  help='Size in bytes above which the log file '
                                  'should rotate', type=int, default=2<<20)
+        self.parser.add_argument('--work-dir', '-w',
+                                 help='Directory used to store all artifacts '
+                                 'created during the process.',
+                                 default='build')
```

Comment:
Due to how unit tests were implemented, this is the only way to test default values. I suggest to open a task to refactor the unit test implementation to start considering default values provided by the config.yaml, as well.